lib/browser/objects-registry.js

```diff
@@ -2,8 +2,12 @@
 
 const v8Util = process.atomBinding('v8_util')
 
-const getOwnerKey = (webContents, contextId) => {
-  return `${webContents.id}-${contextId}`
+const getOwnerKey = (webContents, processId) => {
```

Comment:
You can use a default parameter here:

```js
const getOwnerKey = (webContents, processId = webContents.getProcessId()) {
  return `${webContents.id}-${processId}`
}
```

`webContents.getProcessId()` will be evaluated only if no second argument is provided.